recipe/meta.yaml

```diff
@@ -44,6 +44,7 @@ requirements:
     - libnetcdf  # [not win]
     - libnetcdf 4.6.1  # [win]
     - udunits2
+    - tempest-remap  # [not win]
```

Comment:
@xylar is there a test we can add to ensure that dependency is doing what you expect?
Something similar to what we do with ESMF?

Reply:
It looks like we're currently not testing remapping with ESMF:
https://github.com/conda-forge/nco-feedstock/pull/70/files#diff-50e56299841cca340fa9c08824016349

I can try to add tests or tempest-remap but I'm a bit afraid that whatever problems were caused by these previous tests will come back.

Reply:
Argh... I did not write a comment on why :-/
Probably had a CI issue or something... Thanks for re-enabling that test so we can check it!